templates/include/telemetry/mainPayload.1.schema.json

```diff
@@ -92,18 +67,17 @@
       "type": "object",
       "properties": { }
     },
-    "threadHangStats": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": { }
-      }
-    },
     "UIMeasurements": {
       "type": "array",
       "items": {
         "type": "object",
-        "properties": { }
+        "properties": {
+          "type": {"type": "string"},
+          "action": {"type": "string"},
+          "method": {"type": "string"},
+          "extras": {"type": "string"},
+          "timestamp": {"type": "number"}
+        }
```

Comment:
This looks like an addition rather than a subtraction. Is this an unrelated change, or does this directly support the exclusions mentioned in the description?

Also, is this giving names to the elements of the event array? Is this a solution for letting the jsonschema-transpiler output named structs for tuples?

Reply:
This is an addition, however this is an event _object_, not an array, so it does not solve the tuple issue. (Yes is not technically an exclusion but still needed for production :))